api/app/services/prometheus_client.py

```
@@ -0,0 +1,200 @@
+            return None
+
+        try:
+            value = int(float(result["data"]["result"][0]["value"][1]))
+            logger.info(f"Retrieved total query count from Prometheus: {value}")
+            return value
+        except (IndexError, KeyError, ValueError, TypeError) as e:
+            logger.warning(f"Failed to parse query count from Prometheus: {e}")
+            return None
+
+    async def get_response_time_trend(
+        self, lookback_hours: int = 24
```

Ruff (0.14.1)

Unused method argument: `lookback_hours`

(ARG002)